### PR TITLE
{math}[intel/2020b] EigenExa v2.11

### PR DESCRIPTION
**Pull request by Dankomaister**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/EigenExa/EigenExa-2.11-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+source_urls = ['https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['87dee8ac13f410a007e82df2688fa7f143883229dac729fd20836f4a28fac43d']
+
+preconfigopts = './cleanup && ./bootstrap && '
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['lib/libEigenExa.a'],
```
Comment (jfgrimm) [inline easybuild/easyconfigs/e/EigenExa/EigenExa-2.11-intel-2020b.eb:21]: worth adding a check for the shared library as well
```suggestion
    'files': ['lib/libEigenExa.%s' % x for x in ['a', SHLIB_EXT]],
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/e/EigenExa/EigenExa-2.11-intel-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+description = """EigenExa, a part of KMATHLIB, is a high performance eigen-solver."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['87dee8ac13f410a007e82df2688fa7f143883229dac729fd20836f4a28fac43d']
+
+preconfigopts = './cleanup && ./bootstrap && '
```
[the remark this answers is not in the document]
Comment (Dankomaister) [reply easybuild/easyconfigs/e/EigenExa/EigenExa-2.11-intel-2020b.eb:16]: great! thanks for the suggestion

**Review by Dankomaister — COMMENTED** [review]
Accept suggested changes

**Comment by jfgrimm** [comment]
> Accept suggested changes

Can you update the PR with the changes? You can either use `eb --update-pr 15234 <files-to-update>` or push to the `20220404101947_new_pr_EigenExa211` branch in your fork of easybuild-easyconfigs

**Comment by jfgrimm** [comment]
> Accept suggested changes

Can you update the PR with the changes? You can either use `eb --update-pr 15234 <files-to-update>` or push to the `20220404101947_new_pr_EigenExa211` branch in your fork of easybuild-easyconfigs

**Comment by Dankomaister** [comment]
Okay, I will figure out how to do this.
I'm still a GitHub noob so this is somewhat confusing haha
-sorry

**Comment by Dankomaister** [comment]
done!

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Review by jfgrimm — APPROVED** [review]

**Comment by jfgrimm** [comment]
Going in, thanks @Dankomaister!

**Pull request merged**